editor/src/components/canvas/utopia-canvas-vars.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { CanvasScale, CanvasScrollOffset } from '../../utils/global-positions'
 import { useEditorState } from '../editor/store/store-hook'
 
-export const UtopiaCanvasVarStyleTag = React.memo(() => {
+export const UtopiaCanvasVars = React.memo(() => {
```

Comment:
Almost feels like this should be a hook rather than a component.

Reply:
this will be removed in the next PR